contracts/Zap/ZapMint.sol

```diff
@@ -72,13 +74,13 @@ contract ZapMint is OwnableUpgradeable, ReentrancyGuard {
         require(mapZapMintSupportedTokens[path[0]], "ZapMint: Token not supported for zap minting");
 
         IERC20(path[0]).safeTransferFrom(msg.sender, payable(address(this)), amountIn);
+        IERC20(bdxAddress).safeTransferFrom(msg.sender, payable(address(this)), bdxInMax);
```

Comment:
do we need the `payable` modifier here and in the lina above?

Reply:
nope, removed